packages/sources/ada-balance/src/config.ts

```diff
@@ -4,20 +4,18 @@ import { Config as BaseConfig } from '@chainlink/types'
 export const NAME = 'ADA_BALANCE'
 
 export const DEFAULT_ENDPOINT = 'balance'
-export const DEFAULT_RPC_PORT = 1337
 
 export interface ExtendedConfig extends BaseConfig {
-  isTLSEnabled: boolean
+  httpOgmiosURL: string
+  wsOgmiosURL: string
 }
 
 export const makeConfig = (prefix?: string): ExtendedConfig => {
   const baseConfig = Requester.getDefaultConfig(prefix)
-  baseConfig.api.baseWsUrl = util.getRequiredEnv('WS_API_ENDPOINT', prefix)
-  const port = util.getEnv('RPC_PORT', prefix)
-  baseConfig.rpcPort = port ? parseInt(port) : DEFAULT_RPC_PORT
   baseConfig.defaultEndpoint = DEFAULT_ENDPOINT
   return {
     ...baseConfig,
-    isTLSEnabled: !!util.getEnv('IS_TLS_ENABLED', prefix),
+    httpOgmiosURL: util.getRequiredEnv('HTTP_OGMIOS_URL', prefix),
+    wsOgmiosURL: util.getRequiredEnv('WS_OGMIOS_URL', prefix),
```

Comment:
Can we make this fall back to the old env var, just in case? It should log an error about needing to update if that's the case.

Reply:
If we have good enough backwards compatibility, we could potentially get away with only doing a minor bump

Reply:
Yeah, error out if none of the env vars are set (we're not able to get an HTTP or WS URL for Ogimos)